code/modules/admin/ipintel.dm

```diff
@@ -225,4 +225,68 @@
 		if(vpn_whitelist_add(target_ckey))
 			to_chat(usr, "[target_ckey] was added to the VPN whitelist.")
 		else
-			to_chat(usr, "VPN whitelist unchanged.")
\ No newline at end of file
+			to_chat(usr, "VPN whitelist unchanged.")
+			
+/proc/high_risk_check(var/ip, var/client/C)
+	if(!config.high_risk_checks)
+		return
+	var/ckey = C.ckey
+	if(C.player_age == "Requires database")
+		return
+	if(vpn_whitelist_check(ckey))
+		return 
+	if(C.player_age <= config.high_risk_age)
+		var/list/http[] = world.Export("http://127.0.0.1/hrcheck/check.php?requestedIP=[sanitizeSQL(ip)]") //MAKE SURE PATH IS CORRECT BEFORE MERGING
+
+		if(http)
+			var/status = text2num(http["STATUS"])
```

Comment:
Isnt this lowercase

Reply:
https://github.com/ParadiseSS13/Paradise/blob/6582f22be5f5565a53ee496d9a551b93fdc076f5/code/modules/admin/ipintel.dm#L88